src/krylov.jl

```diff
@@ -92,7 +114,12 @@ function axpby!(
   ::Val{true},
   ::Val{true},
 ) where {T <: Number, Y1 <: Number, Y2 <: Number}
-  y .= x .* s .+ y .* t
```

Comment:
I don't understand how the two `Val` arguments work. Sometimes they're `true` (if `x.simulate_vector` is `true`), and you call `build!()`, but not always.